docs/application/native/guides/ui/dali/capture.md

```diff
@@ -0,0 +1,67 @@
+# Capture
+
+
+Capture snapshot the current DALi scene. The captured result can be saved as a file and also can be used to be rendered by DALi `ImageView` or `ImageVisual`.
+
+In this tutorial, we show how to capture scene and to use it.
+
+
+## Tutorial
+
+Applications should follow the example below to use `Dali::Capture`.
+
+``` C++
+Capture capture = Capture::New();
+
+capture.Start( source, size, path, clearColor, quality);
+```
+
+Each parameter of `Capture::Start` is denoted as follows:
+
+ - **Actor source**: source actor to be rendered scene. Only this source actor and its children are captured.
+ - **Vector2 size**: size of captured result image.
+ - **std::string path**: system path to be saved as a file. If the path is empty, `Dali::Capture` do not save the image as file and the captured result can be used to be rendered by DALi `ImageView` or `ImageVisual`.
+ - **Vector4 clearcolor**: background color of captured scene
+ - **int quality**: a value to control quality of saved image file for jpeg file format in the range [1, 100]
+
+If required, a callback function can be connected to a finished signal:
+
+``` C++
+capture.FinishedSignal().Connect(this, &CaptureSceneExample::OnCaptureFinished);
+
+void CaptureSceneExample::OnCaptureFinished( Capture capture, Capture::FinishState state )
+{
+  if ( state == Capture::FinishState::SUCCEEDED )
+  {
+    // Do something
+  }
+  else
+  {
+    // Do something
+  }
+}
+```
+
+The captured result is also used as a image source to be rendered by DALi.
+
+``` C++
+void CaptureSceneExample::OnCaptureFinished( Capture capture, Capture::FinishState state )
+{
+  if ( state == Capture::FinishState::SUCCEEDED )
+  {
+    std::string url = Toolkit::Image::GenerateUrl( capture.GetNativeImageSource() );
+    Toolkit::ImageView imageView = Toolkit::ImageView::New( url );
+
+    mRootLayer.Add( imageView );
+  }
+  else
+  {
+    // Do something
+  }
+}
+```
+
+## Related Information
+- Dependencies
+  - Tizen 4.0 and Higher for Wearable
+  - Tizen 6.0 and Higher for Mobile
```

Comment:
According to the ACR-1552 and ACR-1553, this update is for the Tizen 6.0

master branch is to handle the updates for Tizen 5.5.
For Tizen 6.0, we use tizen_6.0_prepare branch. 

Would you please change the target branch to tizen_6.0_prepare?

```
  - Tizen 6.0 and Higher for Mobile
  - Tizen 4.0 and Higher for Wearable
```

Also, it looks like capture api for wearable is added since Tizen 6.0.
Would you please check the "Tizen 4.0 and Higher for Wearable" again?

Reply:
Thank you I changed the target branch to tizen_6.0_prepare.
And, yes, these Capture APIs have existed since Tizen 4.0 for Wearable.(Though we added several new APIs in this time)
The two ACR, first one is to copy existing APIs in wearable to mobile profile and, the last one is to add totally new APIs for both profiles.